### PR TITLE
feat: qr scanner optional configuration 

### PR DESCRIPTION
**Pull request by CarolineLCa**
# Summary of Changes

- Placing the scanNow and scanHelp buttons as optional in the `defaultConfiguraiton`

IMG_0121

- Adding translation key for "A valid QR code will scan automatically." message and adjusting error message placing

### Before
IMG_0119

### Now
IMG_0123

- Aligning `contacts list` link on `WhatAreContacts` screen

### Before
IMG_0120

### Now
IMG_0122
# Related Issues

N/A

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [X] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [X] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [X] Updated documentation as needed for changed code and new or modified features;
- [X] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated tests have passed.

_PR template adapted from the Python attrs project._

### CODE REVIEW

File: packages/legacy/core/App/defaultConfiguration.ts [hunk 1]
```diff
@@ -55,4 +55,6 @@ export const defaultConfiguration: ConfigurationContext = {
   supportedLanguages: Object.keys(translationResources) as Locales[],
   showPreface: false,
   disableOnboardingSkip: false,
+  showScanHelp: false,
```
Comment (wadeking98) [inline packages/legacy/core/App/defaultConfiguration.ts]: do you think you could make these true by default so that this isn't a breaking change?

**Review by wadeking98 — COMMENTED** [review]
other than the default values, LGTM

File: packages/legacy/core/App/defaultConfiguration.ts (hunk 1 above)
Comment (CarolineLCa) [reply packages/legacy/core/App/defaultConfiguration.ts]: Done :)

**Comment by CarolineLCa** [comment]
@jleach @bryce-mcmath  may you review please? We have feature blocked by this PR for a few days.

**Review by bryce-mcmath — APPROVED** [review]

**Pull request merged**